### DISCUSSION TITLE
Use own External storage

### DISCUSSION THREAD

**Discussion by SpiderChat** [owncast/owncast#1384] [Q&A]
I have own server. How to use own External storage?

**Comment by gabek** [comment]
You'll have to elaborate.  If it's your own storage on your own server then you don't need to utilize external storage services since you already have local storage you want to use.

If you mean something like you want to run your own Minio S3 service, then you can do that: https://owncast.online/docs/storage/minio/.

**Reply by SpiderChat** [reply]
Can i put my video there then autoplay ?

**Reply by gabek** [reply]
No, Owncast will only distribute content that is ingested via RTMP.

**Reply by SpiderChat** [reply]
Oic. Ok I understand
